sdk/tables/Azure.Data.Tables/tests/TablesRecordedTestSanitizer.cs

```diff
@@ -15,5 +18,27 @@ public override string SanitizeVariable(string variableName, string environmentV
                 _ => base.SanitizeVariable(variableName, environmentVariableValue)
             };
         }
+
+        public override string SanitizeUri(string uri)
+        {
+            var builder = new UriBuilder(base.SanitizeUri(uri));
+            var query = new UriQueryParamsCollection(builder.Query);
+            if (query.ContainsKey(SignatureQueryName))
+            {
+                query[SignatureQueryName] = SanitizeValue;
+                builder.Query = query.ToString();
+            }
+            return builder.Uri.ToString();
```

Comment:
Just call `SanitizeQueryParameters`.  I know this was copy/pasted, but it shouldn't have been done that way originally.